src/main/kotlin/com/recipebook/domain/ingredient/IngredientController.kt

```diff
@@ -0,0 +1,47 @@
+package com.recipebook.domain.ingredient
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.HttpStatus.OK
+import org.springframework.http.ResponseEntity
+import org.springframework.http.ResponseEntity.status
+import org.springframework.stereotype.Controller
+import org.springframework.validation.annotation.Validated
+import org.springframework.web.bind.annotation.*
+import java.util.*
+import javax.servlet.http.HttpServletResponse
+import javax.transaction.Transactional
+
+@CrossOrigin(origins = ["http://localhost:8080", "http://localhost:8080/#"], maxAge = 3600)
+@Controller
+@Validated
+@RequestMapping("/ingredient")
+@Transactional
+/* @Autowired => nie chce przyjąć */
```

Comment:
Autowired nie możesz uzyć na klasie, przeczytaj proszę jak to działa https://www.baeldung.com/spring-autowire
A tą linię możesz usunąć

Reply:
Ok